src/main.ts

```diff
@@ -0,0 +1,75 @@
+import * as core from '@actions/core'
+import * as github from '@actions/github'
+
+const oktokit = github.getOctokit(core.getInput('token'))
+
+async function run() {
+  const owner = github.context.repo.owner
+  const repo = github.context.repo.repo
+  const upstream_owner = core.getInput('upstream_owner')
+  const upstream_repo = core.getInput('upstream_repo')
+  const upstream_branch =
+    core.getInput('upstream_branch') ||
+    (await oktokit.rest.repos.get({owner: upstream_owner, repo: upstream_repo}))
+      .data.default_branch
+  const reviewers = core.getInput('reviewers').split(',')
+  const base_branch = (await oktokit.rest.repos.get({owner, repo})).data
+    .default_branch
+
+  try {
+    const {data: autosync_pulls} =
+      await oktokit.rest.search.issuesAndPullRequests({
+        q: 'label:autosync+state:open+is:pull-request',
+        per_page: 1
+      })
+    if (autosync_pulls.total_count < 1) {
+      const {data: upstream_manifest} = await oktokit.rest.repos.getContent({
+        owner: upstream_owner,
+        repo: upstream_repo,
+        ref: upstream_branch,
+        path: 'src/manifest.json'
```

Comment:
not every repo has this structure eg https://github.com/Mixaill/galaxy-integration-wargaming

We can use optional parameter to this action or do some magic like here: https://github.com/FriendsOfGalaxy/galaxy-integrations-synchronizer/blob/cbf572ed1639813d90596ba37257425544550da9/scripts.py#L192

Reply:
Hmm I see that is definitely something we should consider in the future. But I was also thinking we enforce a light repository structure (e.g. plugin files in `src`) as it is *imo* cleaner and easier to package. And by enforce I don't mean denying plugins or anything like that, just could be as simple as altering the repo structure in the fork. Just an idea tho.